Shops/ShopGoblinTinkerer.cs

```diff
@@ -45,87 +45,71 @@ public void InitShop(string currentShop)
                         shop.item[nextSlot++].SetDefaults(calamity.ItemType("Laudanum"));
                         shop.item[nextSlot++].SetDefaults(calamity.ItemType("StressPills"));*/
                     //}
-                    break;
+                    //break;
                 case "Miscellaneous":
                     if (NPC.downedPirates)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.GoldRing);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         shop.item[nextSlot].SetDefaults(ItemID.DiscountCard);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         shop.item[nextSlot].SetDefaults(ItemID.LuckyCoin);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
```

Comment:
You need `++` otherwise everything will be merged into one slot.

Please review the prices.